### PR TITLE
Bring location finding in

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
As per: https://github.com/praekelt/go-usaid-clinicfinder/blob/develop/src/app.js

Current functionality is just a placeholder. Please transfer code as-is into this repo.

### CODE REVIEW

File: src/ussd_app.js [hunk 1]
```diff
@@ -230,16 +344,238 @@ go.app = function() {
                     " looking for?",
                 ].join("")),
                 choices: [
-                    new Choice("state_end", $("Nearest Clinic")),
-                    new Choice("state_end", $("MMC Clinic")),
-                    new Choice("state_end", $("HCT Clinic")),
+                    new Choice("nearest", $("Nearest Clinic")),
```
Comment (NicolaasDieBaas) [inline src/ussd_app.js:353]: @imsickofmaps Not quite sure what to do with this option as the original clinicfinder code only provided options for MMC and HCT... Does "Nearest Clinic" mean any type of clinic, whichever is closest..?  If I understood the code correctly, choosing the MMC and HCT options do lead to finding the nearest clinic of that type

Comment (imsickofmaps) [inline src/ussd_app.js:353]: @NicolaasDieBaas set both options to `true` in `make_clinic_search_params` utils

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps please review.  not sure if current qa config file will do... it looks ok

File: src/ussd_app.js (hunk 1 above)
Comment (imsickofmaps) [inline src/ussd_app.js:353]: https://github.com/praekelt/go-usaid-mmc/pull/40/files#diff-020cb63d1a6c4bb7c2c5da851dd51bb9R567 should have the following for that option

``` json
"search": {
                "msisdn": "+1234",
                "mmc": "true",
                "hct": "true"
},
```

**Comment by imsickofmaps** [comment]
@NicolaasDieBaas ping me again when you've made the search change and I'll review all together

File: src/ussd_app.js (hunk 1 above)
Comment (NicolaasDieBaas) [inline src/ussd_app.js:353]: @imsickofmaps nice thanks.  that seems easy enough to do.  Will implemented and re-submit for review

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps please review

**Comment by imsickofmaps** [comment]
👍

**Comment by imsickofmaps** [comment]
@NicolaasDieBaas landing for QA deploy

**Pull request merged**